index.js

```diff
@@ -50,6 +50,13 @@ var proxies = [
 ];
 
 /**
+ * Regex to match any comma surrounded by optional whitespace
+ *
+ * @type {RegExp}
+ */
+var ipListSpaceFilter = /\s*,\s*/;
```

Comment:
this is still necessary if there is more than 1 correct? Does this case practically happen? cc @3rd-Eden